docs/content/guidelines/usage.mdx

```diff
@@ -5,74 +5,93 @@ title: Usage guidelines
 import {Caption, Do, Dont} from '@primer/gatsby-theme-doctocat'
 import {Grid, Flex, Box} from '@primer/components'
 
-# Octicons usage guidelines
+## General guidelines
 
-The following guidelines are to be used as a reference for how to use octicons.
+1. Octicons are used to _emphasize_ meaning, not as a replacement.
+2. Adding an octicon is intentional.
+3. An icon is need. Not everything needs an icon: consider a text label before adding an icon.
+4. Octicons are kept at their 1:1 aspect ratio.
```

Comment:
Could we use a more active voice here? For example:

```diff
- Octicons are kept at their 1:1 aspect ratio.
+ Keep octicons at their aspect ratio.
```

Reply:
Yes! Thank you for rewording that. I was struggling with it.